mollie/api/objects/order.py

```diff
@@ -210,6 +212,23 @@ def update_shipment(self, resource_id, data):
         """Update the tracking information of a shipment."""
         return Shipments(self.client).on(self).update(resource_id, data)
 
+    @property
+    def payments(self):
+        try:
+            payments = self["_embedded"]["payments"]
+        except KeyError:
```

Comment:
This assumption is incorrect: the API will not return an empty `payments` object when no data is available, so we can't conclude that someone forgot to set the `embed=payments`.